swarm/storage/chunker.go

```diff
@@ -298,6 +305,13 @@ func (self *TreeChunker) hashChunk(hasher SwarmHash, job *hashJob, chunkC chan *
 	job.parentWg.Done()
 
 	if chunkC != nil {
+		//NOTE: this increases the chunk count even if the local node already has this chunk;
+		//on file upload the node will increase this counter even if the same file has already been uploaded
+		//So it should be evaluated whether it is worth keeping this counter
+		//and/or actually better track when the chunk is Put to the local database
+		//(which may question the need for disambiguation when a completely new chunk has been created
+		//and/or a chunk is being put to the local DB; for chunk tracking it may be worth distinguishing
```

Comment:
We should probably have a cache hit / miss counter for chunks, so we know how many requests we are managing to serve locally, and how many are hitting the network (both Get and Put).

Reply:
My assumption was that `memstore.get` counters are counter for chunks served by the cache.